ECMAScript/core/src/api/rest/AuthAPI.ts

```diff
@@ -1,5 +1,5 @@
-import type { AxiosRequestConfig, AxiosResponse } from 'axios'
-import axios from 'axios'
+import type { AxiosRequestConfig, AxiosResponse } from './axios'
+import { axios } from './axios'
```

Comment:
Could be nice to name this something else like `quiltt-axios` or `retriable-axios` to make it clear that we're wrapping the client with extra things.